examples/google-tiles/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>A-Frame 3D Tiles Component - Basic example</title>
+    <meta name="description" content="Basic example for 3D Tiles component showing google 3d Tiles."></meta>
+    <script src="https://aframe.io/releases/1.5.0/aframe.min.js"></script>
+    <script src="../../dist/aframe-loader-3dtiles-component.js"></script>
+    <!-- <script src="https://unpkg.com/3dstreet@0.4.5/dist/aframe-street-component.js"></script> -->
+
+        <!-- vr teleport controls -->
+    <!-- <script src="https://cdn.jsdelivr.net/npm/aframe-blink-controls@0.4.3/dist/aframe-blink-controls.min.js"></script> -->
+    <style>
+      #guide {
+        position: fixed;
+        top: 0;
+        right: 0;
+        width: 300px;
+        padding: 1rem 2rem;
+        font-family:'Courier New', Courier, monospace;
+        line-height: 1.2;
+        background-color: white;
+        color: black;
+      }
+
+      #guide p {
+        margin-top: 10px;
+      }
+    </style>
+  </head>
+  <body>
+    <a-scene vr-mode-ui="enabled: false" renderer="colorManagement: true;">
+      <a-entity camera="fov:45; near:1; far: 1000" look-controls="reverseMouseDrag: true" wasd-controls="enabled: true" id="camera" cursor-teleport="cameraRig: #camera; cameraHead: #camera;"></a-entity>
+      <a-entity
+        id="tileset"
+        loader-3dtiles="
+          url: https://tile.googleapis.com/v1/3dtiles/root.json; 
+          lat: 37.77522354250163;
+          long: -122.41931773049723;
+          height: -16.5;
+          googleApiKey: INSERTYOURKEYHERE;
```

Comment:
> but there is no option to insert them via the query string.
Correct, the expected behavior in this example is only to place tiles with a long / lat at scene origin 0 0 0. Supporting a user-specified querystring with lat / long / height was not intended to be part of this example.

Reply:
Got it.
It's maybe OK since the example is anyway not exposed in the demo showcase, but if it works well we should probably consider adding it in the future and also make the example usable without editing the code. Maybe after we have occlusion culling and teleport controls?

Reply:
Yes agreed. I've added these tickets to track:
* https://github.com/nytimes/aframe-loader-3dtiles-component/issues/44
* https://github.com/nytimes/aframe-loader-3dtiles-component/issues/43
* https://github.com/nytimes/aframe-loader-3dtiles-component/issues/42